vng_api_common/authorizations/config/models.py

```diff
@@ -0,0 +1,39 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.db import models
+from django.utils.module_loading import import_string
+from django.utils.translation import ugettext_lazy as _
+
+from solo.models import SingletonModel
+from zds_client import Client
+
+
+class AuthorizationsConfig(SingletonModel):
+    api_root = models.URLField(_("api root"), unique=True, default='https://ref.tst.vng.cloud/ac/api/v1')
+    component = models.CharField(_("component"), max_length=50, default='ZRC')
```

Comment:
can we make this a field with `choices`? The components can then be defined in a `DjangoChoices` constant